autoplex/data/common/flows.py

```diff
@@ -237,7 +237,7 @@ def static_run_and_convert(
 
 
 @dataclass
-class DFTStaticMaker(Maker):
+class DFTStaticLabelling(Maker):
```

Comment:
As this maker is now renamed to `DFTStaticLabelling`, we would also need to rename it here on this line https://github.com/YuanbinLiu/autoplex_pub/blob/52731b1cbefd8a4546ee2b769430ff80ffb00868/autoplex/data/common/flows.py#L46